docs/projects.md

```diff
@@ -278,3 +281,96 @@ You should see something like this:
 ```
 
 And, most importantly, your browser should open a page with a jupyter notebook.
+
+
+## Using Github
+There are three main ways to access a local copy of the Class Code, which is stored in the Github repository.  The first way is to navigate to the class Github repository click on the green code button and select 'Download Zip'.  This will provide you with a ZIP file of the class code, which can be run on your personal machine.  Now you might be saying, that is great but I don't want to download a new ZIP file every time something changes.  In that case you want a local copy of the class Github repository.  You should use on the the two following methods to accomplish this.
+
+### Using Github with a Clone
+A clone is a copy of a repository.  You are free to make changes to the repository and will be able to get an up to date version of the original repository.
+To clone the class repository simply click the green **Code** button in the header of the repository and copy the url presented to you for cloning ```https://github.com/tbretl/ae353-sp21.git```. Make sure you have Git installed on your computer (if not view instructions below under Install Git client tools) and then git bash into the directory where you want to store the local copy of the repository.  Then run the following commands.
+```
+git clone https://github.com/tbretl/ae353-sp21.git
+``` 
+The class repository will then be copied into the directory.
+
+To update the class repository you have a two options: ```fetch``` and ```pull```.
+
+* Fetching will retrieve all the new work in the class repository, but will not merge those changes into your local repository.  This means that your work will not be overwritten.  To fetch simply git bash into the repo directory and run the command ```git fetch origin```.
+* After fetching you can run ```git merge main```, where main is the name of the branch.
+* As a shortcut you can run the ```git pull origin``` command which will complete both fetch and merge.
```

Comment:
Does `git pull [repo url]` also work or does this perform the same action as `git pull origin`?

Reply:
This will also work, but it performs the same action as `git pull origin`, because `origin` is defined as the [repo URL].

This can be confirmed by running `git remote -v`.

Reply:
I typically use `git pull origin`, because I can remember origin, but I would hate to have to go and get the repo-URL every time.

Reply:
I understand. That does make it a lot easier.